src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/CatalogPriceRule/Blocks/form.html.twig

```diff
@@ -0,0 +1,98 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(catalogPriceRuleForm) }}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">attach_money</i>
+    {{ 'Catalog price rules'|trans({}, 'Admin.Catalog.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {{ ps.form_group_row(catalogPriceRuleForm.name, {}, {
+        'label': 'Name'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {% if catalogPriceRuleForm.id_shop is defined %}
+        {{ ps.form_group_row(catalogPriceRuleForm.id_shop, {}, {
+          'label': 'Shop'|trans({}, 'Admin.Global')
+        }) }}
+      {% endif %}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.id_currency, {}, {
+        'label': 'Currency'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.id_country, {}, {
+        'label': 'Country'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.id_group, {}, {
+        'label': 'Group'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.from_quantity, {}, {
+        'label': 'From quantity'|trans({}, 'Admin.Catalog.Feature'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.price, {}, {
+        'label': 'Price (tax excl.)'|trans({}, 'Admin.Catalog.Feature'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.leave_initial_price, {}, {'label': ''}) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.from, {}, {
+        'label': 'From'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.to, {}, {
+        'label': 'To'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(catalogPriceRuleForm.reduction_type, {}, {
+        'label': 'Reduction Type'|trans({}, 'Admin.Catalog.Feature'),
```

Comment:
Hey, please write _Reduction type_ without the second capital letter, thank you.